cclient/serialization/json/add_neighbors.c

```diff
@@ -46,6 +46,59 @@ retcode_t json_add_neighbors_serialize_request(
   return ret;
 }
 
+retcode_t json_add_neighbors_serialize_response(
+    const serializer_t *const s, const add_neighbors_res_t *const res,
+    char_buffer_t *out) {
+  retcode_t ret = RC_OK;
+  const char *json_text = NULL;
+  size_t len = 0;
+
+  log_info(json_logger_id, "[%s:%d]\n", __func__, __LINE__);
+
+  cJSON *json_root = cJSON_CreateObject();
+  if (json_root == NULL) {
+    log_critical(json_logger_id, "[%s:%d] %s\n", __func__, __LINE__,
+                 STR_CCLIENT_JSON_CREATE);
+    return RC_CCLIENT_JSON_CREATE;
+  }
+
+  cJSON_AddItemToObject(json_root, "addedNeighbors",
+                        cJSON_CreateNumber(res->added_neighbors));
```

Comment:
what if no neighbors were added?

Reply:
added_neighbors is initialised to 0 in `add_neighbors_res_new`